rasa/server.py

```diff
@@ -593,7 +593,7 @@ async def wrapped() -> None:
 def run_in_thread(f: Callable[..., Coroutine]) -> Callable:
     """Decorator which runs request on a separate thread.
 
-    Some requests (e.g. training or cross-validation) are computional intense requests.
+    Some requests (e.g. training or cross-validation) are computational intense requests.
```

Comment:
```suggestion
    Some requests (e.g. training or cross-validation) are computationally intense requests.  # noqa: E501, W505
```

The comment is required to pass the Code Quality check.

Reply:
I think, instead a line break should be added 😉

Reply:
Oh yeah! For some reason I thought this is the first line of the docs and that always needs to end `.` My bad

Reply:
Done!